packages/http-client/src/http-client.ts

```diff
@@ -532,7 +538,8 @@ async function getCsrfHeaders(
         params: request.params,
         headers: request.headers,
         url: request.url,
-        timeout: request.timeout || defaultResilienceBTPServices.timeout,
+        middleware: request.middleware,
```

Comment:
middleware can't be omitted, so it needs to be always passed as empty list if I don't need it? Is this intentional?